tracer/src/Datadog.Trace.Tools.Runner/GacUninstallCommand.cs

```diff
@@ -39,34 +39,48 @@ private void Execute(InvocationContext context)
             return;
         }
 
-        using var container = NativeMethods.CreateAssemblyCache();
-        var hr = container.AssemblyCache.UninstallAssembly(UninstallAssemblyFlags.None, assemblyName!, IntPtr.Zero, out var position);
+        using var gacMethods = GacNativeMethods.Create();
+        var assemblyCache = gacMethods.CreateAssemblyCache();
 
-        switch (position)
+        var result = Hresult.S_OK;
+        var installedAssemblyNames = gacMethods.GetAssemblyNames(assemblyName!);
```

Comment:
Assembly name isn't versioned, right? Should this be trying to uninstall a _specific_ version?

Reply:
that line, returns the FullAssemblyName from a Normal name, I'm doing like this to no change the behaviour of the command. I tried to add full assembly name support but I was unable to use that full name with the Uninstall method. How do you do it in the FleetInstaller?

Reply:
We register the path to the native loader using `FUSION_INSTALL_REFERENCE.FUSION_REFCOUNT_FILEPATH_GUID` when we GAC install `Datadog.Trace.dll`

And we can add multiple versions IIRC 🤔 
This is our implementation of this:
https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/src/Datadog.FleetInstaller/GacInstaller.cs

And these are the test cases we check:
https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/test/Datadog.Trace.IntegrationTests/FleetInstaller/GacInstallerTests.cs

We may not _want_ the same behaviour here, but we should probably make the cases are compatible at least...